kornia/contrib/extract_patches.py

```diff
@@ -8,22 +8,44 @@
 
 from kornia.core import Module, Tensor, pad
 
+FullPadType = Tuple[int, int, int, int]
```

Comment:
```suggestion
FullPadType = tuple[int, int, int, int]
```
can we use `from __future__ import annotations` ?

Reply:
This apparently doesn't work for a TypeAlias in 3.8, I'm not sure why (so this line doesn't get accepted by mypy). I'd rather keep everything with `Tuple` rather than some things `Tuple` and some `tuple`.